### DISCUSSION TITLE
[V5] Various errors after upgrading dependencies

### DISCUSSION THREAD

**Discussion by ArthurHetem** [adonisjs/core#2435] [Help] [answered]
I upgraded my dependencies today and i got several errors when i started the server, here is the terminal 
`(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning
(Use `node --trace-warnings ...` to show where the warning was created)
(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning
(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning
(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning
(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning
(node:39858) container.with() is deprecated. Use container.withBindings() instead: DeprecationWarning

IocLookupException: E_IOC_LOOKUP_FAILED: Cannot resolve "Adonis/Core/BodyParserMiddleware" namespace from the IoC Container

at /home/arthur/joguinho/backend/start/kernel.ts(anonymous):23
18  |
19  | An array of global middleware, that will be executed in the order they
20  | are defined for every HTTP requests.
21  |
22  */
23  Server.middleware.register(['Adonis/Core/BodyParserMiddleware'])
24  
25  /*
26  |--------------------------------------------------------------------------
27  | Named middleware
28  |--------------------------------------------------------------------------

1 Function.lookupFailed
  /home/arthur/joguinho/backend/node_modules/@adonisjs/fold/build/src/Exceptions/IocLookupException.js:18

2 Ioc.lookupOrFail
  /home/arthur/joguinho/backend/node_modules/@adonisjs/fold/build/src/Ioc/index.js:254

3 IocResolver.resolve
  /home/arthur/joguinho/backend/node_modules/@adonisjs/fold/build/src/Resolver/index.js:80

4 MiddlewareStore.resolveMiddleware
  /home/arthur/joguinho/backend/node_modules/@adonisjs/http-server/build/src/MiddlewareStore/index.js:62

5 MiddlewareStore.register
  /home/arthur/joguinho/backend/node_modules/@adonisjs/http-server/build/src/MiddlewareStore/index.js:69

6 Module._compile
  /home/arthur/joguinho/backend/node_modules/pirates/lib/index.js:99

[ warn ]  Underlying HTTP server died with "0 code"
[ info ]  watching file system for changes`

Anyone have any ideas on whats going on?

Thanks

**Comment by thetutlage** [comment, marked as answer]
Upgrading dependencies without release notes shouldn't be expected to work. I recommend sticking to the existing versions and wait for us to announce the release

**Reply by ArthurHetem** [reply]
Ok, i downgraded, thanks and sorry about that.
